packages/server-ct/crossword-example/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const VueLoaderPlugin = require('vue-loader/lib/plugin')
```

Comment:
Can we delete this file and instead use Vue CLI in the cypress/plugin file? It’s confusing because vue.config.js isn’t being used